platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/creation/tipset/TipsetEventCreator.java

```diff
@@ -483,7 +483,7 @@ public void clear() {
     public String toString() {
         final StringBuilder sb = new StringBuilder();
         sb.append("Minimum generation non-ancient: ")
```

Comment:
Now might be a good time to update this out-of-date field label

Reply:
fixed